components/item/AwesomeSwiper.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <swiper
+    ref="awesome"
+    class="swiper"
+    :options="swiperOptions"
+  >
+    <swiper-slide
+      v-for="(item, index) in media"
+      :key="index"
+    >
+      <MediaCard
+        :europeana-identifier="item.europeanaIdentifier"
+        :image-src="item.about"
+      />
+    </swiper-slide>
+    <div
+      slot="pagination"
+      class="swiper-pagination"
+    />
+    <div
+      slot="button-prev"
+      class="swiper-button-prev"
+    />
+    <div
+      slot="button-next"
+      class="swiper-button-next"
+    />
+  </swiper>
+</template>
+
+<script>
+  import { Swiper, SwiperSlide, directive } from 'vue-awesome-swiper';
+  import 'swiper/css/swiper.css';
+  import MediaCard from './MediaCard';
+
+  export default {
+    name: 'AwesomeSwiper',
+    components: {
+      Swiper,
+      SwiperSlide,
+      MediaCard
+    },
+    directives: {
+      swiper: directive
+    },
+    props: {
+      europeanaIdentifier: {
+        type: String,
+        required: true
+      },
+      media: {
+        type: Array,
+        required: true
+      }
+    },
+    data() {
+      return {
+        swiperOptions: {
+          slidesPerView: 2,
+          spaceBetween: 10,
+          centeredSlides: true,
+          navigation: {
+            nextEl: '.swiper-button-next',
+            prevEl: '.swiper-button-prev'
+          },
+          pagination: {
+            el: '.swiper-pagination',
+            clickable: true
+          }
+        }
+      };
+    },
+    computed: {
+      swiper() {
+        return this.$refs.awesome.$swiper;
+      }
+    },
+    mounted() {
+      // console.log('Current Swiper instance object', this.swiper)
+      // this.swiper.slideTo(3, 1000, false)
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+  /* .swiper-slide {
```

Comment:
let's get rid of the unused css.

Reply:
sure, this was on the example but I didn't see the purpose of it so had commented out, will delete